k9mail/src/main/java/com/fsck/k9/activity/license/OssLicenseViewActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.fsck.k9.activity.license;
+
+import android.content.res.AssetManager;
+import android.graphics.Typeface;
+import android.os.Bundle;
+import android.text.util.Linkify;
+import android.view.View;
+import android.widget.ScrollView;
+import android.widget.TextView;
+
+import com.fsck.k9.activity.K9Activity;
+
+import org.apache.commons.io.IOUtils;
+
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Activity shows third party software license.
```

Comment:
If it was called ThirdPartyLicenseViewActivity, this would be repeating the class name. As it is it adds little except mild confusion.